docs/Chef-CLI-Catalog.md

```diff
@@ -0,0 +1,91 @@
+# Chef CLI (Catalog)
+
+Throughout the history of Chef, we have created and acquired a variety
+of tools that have grown to a point that, as a Chef practitioner, it is
+hard to know, discover and understand all of them. A few of the tools
+that our users use (almost) every day are:
```

Comment:
I think having a catalog of all available commands/tools is very important. Once the user becomes aware of a tool, the next step is really for them to figure out what it does and if it's useful to them. Is that part of the intended scope of this work? If so then we can collaborate w/ UX team on figuring where/how to display relevant information about each tool and maybe include URLs to more detailed information (e.g. docs page) to users

Reply:
Yes, this is exactly one of the ideas/goals and therefore, collaboration with UX will be required.